tests/aws/services/kinesis/conftest.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+import pytest
+
+from localstack.utils.common import poll_condition
+
+LOG = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def register_kinesis_consumer(aws_client):
+    kinesis = aws_client.kinesis
+    consumer_arns = []
+
+    def _register_kinesis_consumer(stream_arn: str, consumer_name: str):
+        response = kinesis.register_stream_consumer(
+            StreamARN=stream_arn, ConsumerName=consumer_name
+        )
+        consumer_arn = response["Consumer"]["ConsumerARN"]
+        consumer_arns.append(consumer_arn)
+
+        return consumer_arn
```

Comment:
Suggestion: it's usually more useful to return the response from the API call rather than a piece of data from within it. That way if there are other properties that are needed, they are available. See for example https://github.com/localstack/localstack/blob/0b060540fcd5a9826fcdf7a17aace861f6ea76d9/localstack/testing/pytest/fixtures.py#L372-L389 vs https://github.com/localstack/localstack/blob/0b060540fcd5a9826fcdf7a17aace861f6ea76d9/localstack/testing/pytest/fixtures.py#L614-L631

Reply:
fixed in: https://github.com/localstack/localstack/commit/7580ee14132c9696bf976feace0df8bc60523fcb